mep/people/management/commands/export_members.py

```diff
@@ -22,7 +22,7 @@ class Command(BaseExport):
     model = Person
 
     csv_fields = [
-        "uri",
```

Comment:
We need to keep the uri for the member export for backwards compatibility.

I'm not sure "slug" is meaningful to an outside audience...  Inclined to use URIs for creators even though they don't resolve, but I don't feel strongly about that. Do you want to use slug for creators export with a different label?

Reply:
on second thought - because there's overlap between members and creators, I recommend we stick with URIs so the two data exports can easily be used together

Reply:
Should I make my own URIs for the creators then / those that don't resolve, using the slug?

Reply:
ah! I'd forgotten that we don't generate urls for persons without accounts (and it wouldn't make sense to create member uris for them anyways)

is it weird to include a member uri in the creator export if the person is a member?  

I think that yes, we should create uris for creators - this means we should create a super-minimal /creator/slug/ view - maybe it could return brief json response only? (I feel like I did this somewhere else for URIs that were only referenced in a data export... I looked quickly at mep-django urls and didn't see it.)

The additional value of a view like that it will give us a way to check for renamed slugs, assuming we adapt the same redirect logic for old slugs that we have for persons.

However, it does increase the work required to complete this export! I'd be ok with slug (but label as id) for creators.